src/main/resources/brp-api-gezag.yaml

```diff
@@ -0,0 +1,199 @@
+          TijdelijkGeenGezag: "#/components/schemas/TijdelijkGeenGezag"
+          GezagNietTeBepalen: "#/components/schemas/GezagNietTeBepalen"
+    GezagOuder:
+      type: object
+      properties:
+        burgerservicenummer:
+          $ref: "#/components/schemas/Burgerservicenummer"
+      required:
+        - burgerservicenummer
+    Minderjarige:
+      type: object
+      properties:
+        burgerservicenummer:
+          $ref: "#/components/schemas/Burgerservicenummer"
+      required:
+        - burgerservicenummer
+    TweehoofdigOuderlijkGezag:
+      allOf:
+        - $ref: "#/components/schemas/AbstractGezagsrelatie"
+        - type: object
+          properties:
+            ouders:
+              type: array
+              items:
+                $ref: "#/components/schemas/GezagOuder"
+              minItems: 2
+              maxItems: 2
+            minderjarige:
+              $ref: "#/components/schemas/Minderjarige"
+    EenhoofdigOuderlijkGezag:
+      allOf:
+        - $ref: "#/components/schemas/AbstractGezagsrelatie"
+        - type: object
+          properties:
+            ouder:
+              $ref: "#/components/schemas/GezagOuder"
+            minderjarige:
+              $ref: "#/components/schemas/Minderjarige"
+    Meerderjarige:
+      type: object
+      properties:
+        burgerservicenummer:
+          $ref: "#/components/schemas/Burgerservicenummer"
+    GezamenlijkGezag:
+      allOf:
+        - $ref: "#/components/schemas/AbstractGezagsrelatie"
+        - type: object
+          properties:
+            ouder:
+              $ref: "#/components/schemas/GezagOuder"
+            derde:
+              $ref: "#/components/schemas/Meerderjarige"
+            minderjarige:
+              $ref: "#/components/schemas/Minderjarige"
+    Voogdij:
+      allOf:
+        - $ref: "#/components/schemas/AbstractGezagsrelatie"
+        - type: object
+          properties:
+            derden:
+              type: array
+              items:
+                $ref: "#/components/schemas/Meerderjarige"
+              minItems: 0
+              maxItems: 2
+            minderjarige:
+              $ref: "#/components/schemas/Minderjarige"
+    TijdelijkGeenGezag:
+      allOf:
+        - $ref: "#/components/schemas/AbstractGezagsrelatie"
+    GezagNietTeBepalen:
+      allOf:
+        - $ref: "#/components/schemas/AbstractGezagsrelatie"
+    Gezag:
+      type: array
+      items:
+        $ref: "#/components/schemas/AbstractGezagsrelatie"
+      minItems: 0
+    Persoon:
+      type: object
+      description: |
+        * **geslacht** - wordt gevuld met waarden voor 'Geslacht' in 'tabelwaarden.csv'.
+      properties:
+        burgerservicenummer:
+          $ref: '#/components/schemas/Burgerservicenummer'
+        gezag:
+          $ref: "#/components/schemas/Gezag"
+        toelichting:
+          type: object
+          properties:
+            type:
+              type: string
```

Comment:
Patric, heb je een voorbeeld, of overzicht van wat hierin komt? Moet ik ook weten omdat dit in alle features moet worden opgenomen (dus voorspelbaar zijn wat hier komt te staan)